<launch>

	<node name="visualizer" pkg="visualizer" type="visualizer_node" output="screen"> 
		<param name="topic_gaussian" type="string" value="/camera/rgb/image_gaussian_raw"/>
		<param name="topic_resized" type="string" value="/camera/rgb/image_resized_raw"/>
	</node>

</launch>
